senders/client_factory.go

```diff
@@ -2,11 +2,12 @@ package senders
 
 import (
 	"fmt"
-	"github.com/wavefronthq/wavefront-sdk-go/internal"
 	"net/url"
 	"os"
 	"strconv"
 	"strings"
+
+	"github.com/wavefronthq/wavefront-sdk-go/internal"
```

Comment:
Can we revert this change?

Reply:
Generally with go, standard library imports are listed first followed by a space and then 3rd party imports.